src/streamlink_cli/argparser.py

```diff
@@ -467,6 +467,15 @@ def boolean(value):
 
 stream = parser.add_argument_group("Stream options")
 stream.add_argument(
+    "--url",
+    dest="url_param",
+    metavar="URL",
+    help="""
+    Alternative parameter to specify a URL to attempt to
```

Comment:
How about this version? :)

I agree about the config guides/etc. there are a lot of options, and the specific behaviour of those options is not exactly well documented :)